precompiles/peaq-did/src/lib.rs

```diff
@@ -104,8 +112,14 @@ where
 			Some(caller).into(),
 			peaq_pallet_did::Call::<Runtime>::add_attribute {
 				did_account: did_account_addr,
-				name: name.as_bytes().to_vec(),
-				value: value.as_bytes().to_vec(),
+				name: BoundedVec::<u8, <Runtime>::BoundedDataLen>::try_from(
```

Comment:
Should we use other instead of the unwrap? (Maybe unwrap_or_else before)? Because it looks try_from may fail